<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>930</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="itemList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>891</width>
     <height>441</height>
    </rect>
   </property>
   <row>
    <property name="text">
     <string>456</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>123</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="previousPage">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>530</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Предыдущая</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextPage">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>530</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Следующая</string>
   </property>
  </widget>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>30</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Пользователи</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="onlyOnline">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>530</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>только online</string>
   </property>
  </widget>
  <widget class="QLabel" name="pageNum">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>540</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
